src/test/java/org/broadinstitute/hellbender/tools/copynumber/plotting/PlotDenoisedCopyRatiosIntegrationTest.java

```diff
@@ -0,0 +1,167 @@
+    private static final File COPY_RATIOS_WITH_SAMPLE_NAME_MISMATCH_FILE = new File(TEST_SUB_DIR, "plotting-copy-ratios-with-sample-name-mismatch.tsv");
+    private static final File COPY_RATIOS_WITH_MISSING_INTERVALS_FILE = new File(TEST_SUB_DIR, "plotting-copy-ratios-with-missing-intervals.tsv");
+    private static final File COPY_RATIOS_OUT_OF_DICTIONARY_BOUNDS_FILE = new File(TEST_SUB_DIR, "plotting-copy-ratios-out-of-dictionary-bounds.tsv");
+
+    private static final String OUTPUT_PREFIX = "test";
+    private static final int THRESHOLD_PLOT_FILE_SIZE_IN_BYTES = 50000;  //test that data points are plotted (not just background/axes)
+
+    //checks that output files with reasonable file sizes are generated, but correctness of output is not checked
+    @Test
+    public void testPlotting() {
+        final File outputDir = createTempDir("testDir");
+        final String[] arguments = {
+                "-" + CopyNumberStandardArgument.STANDARDIZED_COPY_RATIOS_FILE_SHORT_NAME, STANDARDIZED_COPY_RATIOS_FILE.getAbsolutePath(),
+                "-" + CopyNumberStandardArgument.DENOISED_COPY_RATIOS_FILE_SHORT_NAME, DENOISED_COPY_RATIOS_FILE.getAbsolutePath(),
+                "-" + StandardArgumentDefinitions.SEQUENCE_DICTIONARY_SHORT_NAME, SEQUENCE_DICTIONARY_FILE.getAbsolutePath(),
+                "-" + StandardArgumentDefinitions.OUTPUT_SHORT_NAME, outputDir.getAbsolutePath(),
+                "-" + CopyNumberStandardArgument.OUTPUT_PREFIX_SHORT_NAME, OUTPUT_PREFIX
+        };
+        runCommandLine(arguments);
+        Assert.assertTrue(new File(outputDir, OUTPUT_PREFIX + ".denoisedLimit4.png").exists());
+        Assert.assertTrue(new File(outputDir, OUTPUT_PREFIX + ".denoisedLimit4.png").length() > THRESHOLD_PLOT_FILE_SIZE_IN_BYTES);
+        Assert.assertTrue(new File(outputDir, OUTPUT_PREFIX + ".denoised.png").exists());
+        Assert.assertTrue(new File(outputDir, OUTPUT_PREFIX + ".denoised.png").length() > THRESHOLD_PLOT_FILE_SIZE_IN_BYTES);
+        Assert.assertTrue(new File(outputDir, OUTPUT_PREFIX + ".standardizedMAD.txt").exists());
+        Assert.assertTrue(new File(outputDir, OUTPUT_PREFIX + ".standardizedMAD.txt").length() > 0);
+        Assert.assertTrue(new File(outputDir, OUTPUT_PREFIX + ".denoisedMAD.txt").exists());
+        Assert.assertTrue(new File(outputDir, OUTPUT_PREFIX + ".denoisedMAD.txt").length() > 0);
+        Assert.assertTrue(new File(outputDir, OUTPUT_PREFIX + ".deltaMAD.txt").exists());
+        Assert.assertTrue(new File(outputDir, OUTPUT_PREFIX + ".deltaMAD.txt").length() > 0);
+        Assert.assertTrue(new File(outputDir, OUTPUT_PREFIX + ".scaledDeltaMAD.txt").exists());
+        Assert.assertTrue(new File(outputDir, OUTPUT_PREFIX + ".scaledDeltaMAD.txt").length() > 0);
+        final double standardizedMAD = ParamUtils.readValuesFromFile(new File(outputDir, OUTPUT_PREFIX + ".standardizedMAD.txt"))[0];
```

Comment:
Are these MAD numbers akin to the old qc metric?

Reply:
It really does make some of our collaborator's lives easier to have all of those....

Reply:
No problem.  I think it might make sense to move the emission of these to DenoiseReadCounts, eventually.